test/phlex/view/text.rb

```diff
@@ -38,4 +38,17 @@ def template
 			expect(output).to be == "2.0"
 		end
 	end
+
+	with "deprecated text method" do
+		view do
+			def template
+				text "Depreacted"
+			end
+		end
+
+		it "renders content and prints deprecation warning" do
+			expect(example).to receive(:warn).with("DEPRECATED: The `text` method has been deprecated in favour of `plain`. Please use `plain` instead. The `text` method will be removed in a future version of Phlex. Called from: test/phlex/view/text.rb:47:in `template'")
```

Comment:
Seems like it's super hard to check if something gets printed to STDERR with Sus. At least I haven't figured out how to make it pass in CI. Also the matchers doesn't show a diff why the assertion failed 🤔  This test passes locally though.

If you have any idea, let me know